source/models/base_model.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Offers base class, Model, from which to derive various binary classifiers,
+embedding-bassed or otherwise (e.g. TFIDF).
+
+June, 2019
+@author: Joshua Rubin
+"""
+
+class Model:
+    """ Base class for binary classifier to identify fraudulent tweet . """
+   
+    def __init__(self, corpus=None, embedded_corpus=None):
```

Comment:
Why have an `__init__` here is it just passes?

Reply:
Just a loose end - will take out.  Was trying to define a standard interface for models... but not 100% sure how best to do that with Python classes.